content/basics/use-ably.textile

```diff
@@ -0,0 +1,119 @@
+
+The Ably REST interface is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h2(#rest-api). Raw REST API
+
+Ably also provides a raw REST API where you don’t need to install any SDKs to use. Ably recommends using the REST library where possible but you may want to use the raw REST API if:
+
+* Your chosen platform isn't supported
+* You have resourcing constraints
+ 
+If you decide to use the REST API directly there is "extensive documentation":/api/rest-api to support you.
+
+h2(#protocol-adapters). Protocol Adapters
+
+The Ably platform also provides "protocol adapters":/protocols, providing open protocol support and interoperability with a huge array of third party client libraries supporting every imaginable platform.
+
+Although Ably SDKs are the recommended method for connecting to Ably, we also provide a number of protocol adapters as an alternative method or if Ably doesn't provide an SDK for that platform. 
+
+The advantage to protocol adapters is that they require fewer resources and network overhead, however they do not support the full set of Ably features.
+
+We offer protocol adapters for:
+
+* "External MQTT libraries":#mqtt
+* "Ably SSE and EventSource API":#sse-eventsource
+* "Pusher and PubNub protocols":#pusher-pubnub
+
+h3(#mqtt). External MQTT libraries
+
+The Ably "MQTT protocol adapter":/protocols/mqtt translates between "MQ Telemetry Transport (MQTT)":/https://mqtt.org and Ably’s own protocol, allowing for seamless integration of any systems you may have. MQTT is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where lightweight, low-bandwidth networks are key to having effective communication between various devices.
+
+CAVEAT-INPUT 
+ Ably recommends you use MQTT to interact with Ably when:
+
+ * Using devices which do not have a device-specific Ably SDK, such as Arduino platforms, C/C++ applications, and so on
+ * If bandwidth is limited and you want to keep network traffic to a minimum
+
+Read more in our "MQTT section":/protocols/mqtt.
+
+h3(#sse-eventsource). Ably SSE and EventSource API
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime interface, or even an MQTT library, is impractical. 
+
+HTTP streaming allows for requests from clients to be held by a server, allowing it to push data to the client without further requests. This helps avoid the overhead involved in normal HTTP requests. Server-sent events (SSE) provide a thin layer on top of HTTP streaming. A common use of SSE is through the use of the EventSource API in all modern web browsers.
+
+Read more in our "SSE section":/protocols/sse.
+
+CAVEAT-INPUT 
```

Comment:
This page uses a lot of caveat boxes. This, to me, is a sign that this is important information that should be in the content. Caveat boxes should be used fairly sparingly.